pages/linux/pacstall.md

```diff
@@ -0,0 +1,36 @@
+# pacstall
+
+> Pacstall, making easier to install programs.
```

Comment:
I'm not sure if `pacstall` includes AUR repositories, its repo says it related to AUR.
```suggestion
> Pacstall, make it easier to install AUR packages for Ubuntu.
```